lib/Ama/Controller/Emailfeedback.pm

```diff
@@ -0,0 +1,26 @@
+package Ama::Controller::Emailfeedback;
+use Mojo::Base 'Mojolicious::Controller';
+
+sub submit{
+    my $self = shift;
+    my $sendgrid = $self->sendgrid;
+    my $feedback_comment = $self->param('feedback_comment');
+    #$c -> feedback -> submit($feedback);
+    warn $self -> req -> body;
+    my $send = $sendgrid -> mail(
+        text => $feedback_comment,
+        to      => q(ajin@keystone-it.com),
+        cc      => q(jsiems@keystone-it.com),
+        bcc     => q(bmeyer@keystone-it.com),
+        from    => q(ajin@keystone-it.com),
+        subject => "feedback received from AMA",)->send;
+    $self -> emailfeedback->add($feedback_comment);
```

Comment:
This is a great case where the name emailfeedback isn't so great.  emailfeedback is the model here.  Seems weird to call a DB model via a method with the name email in it.  The emailfeedback model is doing nothing with email.